assets/src/js/commands/admin-commands.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Admin Commands
+ *
+ * Core WordPress commands for Secure Custom Fields administration.
+ * This file registers navigation commands for all primary SCF admin screens,
+ * enabling quick access through the WordPress commands interface (Cmd+K / Ctrl+K).
+ *
+ * @since 6.5.0
+ */
+
+wp.domReady( () => {
+	if ( ! wp.data?.dispatch?.( 'core/commands' ) || ! acf?.data ) {
+		return;
+	}
+
+	const { __ } = wp.i18n;
+	const { createElement } = wp.element;
+	const { Icon } = wp.components;
+	const commandStore = wp.data.dispatch( 'core/commands' );
```

Comment:
It is possible to use import statements to let the build tackle detecting all dependencies. @cbravobernal added all necessary code to automate it. See:
- https://github.com/WordPress/secure-custom-fields/pull/86